microsoft-edge/web-platform/user-agent-string.md

```diff
@@ -9,23 +9,28 @@ ms.prod: microsoft-edge
 keywords: microsoft edge, compatibility, web platform, user agent string, ua string, ua overrides
 ---
 
-# Microsoft Edge's User Agent String (Desktop)
+# Microsoft Edge User Agent String (Desktop)  
 
-A browser's user agent (UA) string can be used to detect what version of a specific browser is being used on a certain operating system. Like other browsers, Microsoft Edge includes this information in the `User-Agent` HTTP header whenever it makes a request to a site. It can also be accessed via JavaScript by querying the value of `navigator.userAgent`.
+A browsers user agent \(UA\) string is able to be used to detect what version of a specific browser is being used on a certain operating system.  Like other browsers, Microsoft Edge includes this information in the `User-Agent` HTTP header whenever it makes a request to a site.  It may also be accessed via JavaScript by querying the value of `navigator.userAgent`.  

```

Comment:
@jm-trd-ms I think this should still be "browser's" since its possessive.

Reply:
The user agent string \(UA string\) for a browser...